docs/deploy_features.md

```diff
@@ -321,6 +322,13 @@ For an alternate frontend that's written in Web Components and deployed to Stati
 on [using a different backend](https://github.com/Azure-Samples/azure-search-openai-javascript#using-a-different-backend).
 Both these repositories adhere to the same [HTTP protocol for AI chat apps](https://aka.ms/chatprotocol).
 
+## Enabling query rewriting
+
+By default, [query rewriting](https://learn.microsoft.com/azure/search/semantic-how-to-query-rewrite) from the search service is not enabled. To enable query rewriting, set the following environment variables:
```

Comment:
I wonder if we should clarify in-repo query rewriting vs search service query rewriting here, given how often I refer to the query rewriting stage of this repo.

"Note that the search service query rewriting feature is different from the query rewriting step that is used by the Chat tab in the codebase. That query rewriting step also incorporates conversation history, while the search service query rewriting step only considers the query itself."

?

Reply:
this is accurate, taking your suggestion